WebDriver.io/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "performance-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/.bin/wdio wdio.conf.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "assert": "^1.4.1",
```

Comment:
this is a native node API (https://nodejs.org/dist/latest-v10.x/docs/api/assert.html) no need to add it as dependency